projects/control-service/projects/job-builder-secure/Dockerfile.python.vdk

```diff
@@ -51,7 +51,9 @@ RUN : \
           && yum autoremove build-essential gcc glibc-devel git unzip -y;  fi \
     && echo "Installing native dependencies ..." \
     && yum install libffi-devel libstdc++ findutils openssl-c_rehash -y \
-    && echo "Refreshing CA certificates ..." \
+    && echo "Downloading AWS RDS certificate bundle..." \
+    && wget https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem -O /etc/ssl/certs/global-bundle.pem \
```

Comment:
I think the best place for this installation is in the job-base-image-secure since it is a relatively small file, and we will avoid the dependency on AWS with each job build.